posthog/hogql/grammar/README.md

```diff
@@ -0,0 +1,49 @@
+## How to generate source code files from grammar
+
+Grammar is located inside `HogQLLexer.g4` and `HogQLParser.g4` files.
+
+To generate source code you need to install locally the `antlr4` binary:
+
+```bash
+brew install antlr
+```
+
+or this piece of art if you're using bash on ubuntu for quick access:
+
+```bash
+export ANTLR_VERSION=4.11.1
+
+sudo apt-get install default-jre
```

Comment:
Same situation as in CI – curious why not `apt install antlr4`?

Reply:
Thanks for the review! Resolved all points I hope